service/src/test/java/uk/nhs/adaptors/gp2gp/IdGenerator.java

```diff
@@ -0,0 +1,9 @@
+package uk.nhs.adaptors.gp2gp;
+
+import java.util.UUID;
+
+public class IdGenerator {
```

Comment:
A recent PR introduced RandomIdGeneratorService. This is preferred since the generated ID can be mocked for testing.